### DISCUSSION TITLE
script 작성 시에 paramter 값을 불러서 요청마다 다르게 보낼 수 있나요?

### DISCUSSION THREAD

**Discussion by yeji-tomato** [naver/ngrinder#898] [Q&A]
예를 들어 요청을 보내고자 할 때 
paramter 값을 다른 파일이나 URL을 통해 1번 요청을 보낼 시에  
1번째 값을 읽고 2번째 요청을 보낼 시에 2번째 값을 읽게 작성하는 방법이 있을까요?

**Comment by yeji-tomato** [comment]
resources 폴더에 파일 업로드 해서 스크립트 코드에 해당 파일을 읽어서 random 함수로 문제 해결했습니다!
https://github.com/naver/ngrinder/wiki/How-to-use-resources

**Comment by donggyu04** [comment]
아니면 간단하게 스크립트 내부에 테스트용 파라미터들을 배열등과 같은 자료구조에 담아 두시고, 매 요청시마다 랜덤 값을 추출하여 추출된 랜덤 값과 맵핑되는 파라미터를 뽑아서 사용하게 하셔도 됩니다.
